lbrynet/core/utils.py

```diff
@@ -94,14 +94,29 @@ def obfuscate(plain):
     return base64.b64encode(plain).encode('rot13')
 
 
-def check_connection(server="lbry.io", port=80):
+def check_connection(server="lbry.io", port=80, timeout=2, bypass_dns=False):
     """Attempts to open a socket to server:port and returns True if successful."""
+    log.debug('Checking connection to %s:%s', server, port)
     try:
-        log.debug('Checking connection to %s:%s', server, port)
-        host = socket.gethostbyname(server)
-        s = socket.create_connection((host, port), 2)
+        if not bypass_dns:
```

Comment:
I don't think there should be a `bypass_dns` parameter since the exception is handled regardless